teacher.rb

```diff
@@ -0,0 +1,10 @@
+class Teacher < Person
+  def initialize(specialization, age, name = 'Unkown', parent_permission = 'true')
+    super(age, name, parent_permission)
```

Comment:
- Let's also do the same here as we only need the name and age of the teacher as explained in the previous review.